src/index.html

```diff
@@ -6,9 +6,93 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+     href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet"
+    href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav-container">
+      <a
+      href="#home"
+      class="homeLink">
+        <img
+        src="./images/logo.png"
+        alt="logo">
+      </a>
+      <nav>
+        <ul class="nav-list">
+          <li>
+            <a
+            href="#Apple"
+            class="is-active nav-link"
+            >
+            apple
+            </a>
+          </li>
```

Comment:
```suggestion
            <a
              href="#Apple"
              class="is-active nav-link"
            >
              Apple
            </a>
          </li>
```

Check the example from the checklist again, 
add a 2-space indentation for attributes

Also, please fix this comment: 
<img width="714" alt="Screenshot 2023-06-09 at 19 34 38" src="https://github.com/mate-academy/layout_moyo-header/assets/32853034/51c82a31-a253-429b-b0fd-314c645e5dc1">